test/method/testmoments.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Test the Moments method in cclib"""
+
+from __future__ import print_function
+
+import sys
+import unittest
+
+import numpy
+from numpy.testing import assert_almost_equal
+
+from cclib.method import Moments
+from cclib.parser import GAMESS, Gaussian
+
+sys.path.insert(1, "..")
+
+from ..test_data import getdatafile
+
+
+class MomentsTest(unittest.TestCase):
+    def test_results(self):
+        data, _ = getdatafile(Gaussian, "basicGaussian16", ["water_mp2.log"])
+        x = Moments(data).calculate()
+        assert_almost_equal(x[0], [0, 0, 0], 5)
+        assert_almost_equal(x[1], [0, 0, -0.91543], 5)
+        assert_almost_equal(x[2][0] + x[2][3] + x[2][5], 0)
+
+    def test_origin_displacement(self):
+        data, _ = getdatafile(Gaussian, "basicGaussian16", ["water_mp2.log"])
+        x = Moments(data).calculate()
+        y = Moments(data).calculate(origin=(2e7,18,28))
```

Comment:
Can you make sure all of the lines in the test file follow PEP 8? The method itself looks ok.